### ISSUE TITLE
SpeedGoat io

### ISSUE THREAD

**Issue by lindbloom** [Huh/PopR_SDGFP#211]
when I click on the SD image going across the bottom of the home page, it takes me back to the top of the page.  When I click in the SD link under login dropdown menu, it takes me to the old popr and it would probably be good to switch this to new popr

**Comment by Huh** [comment]
Agreed.  My intent was to create a development site SD_DEV that was a place
where we can implement and test new features.  Once that site is approved
it will be moved over to the old address and so the link will work as
expected.  Would you like me to replace the old site with the new site?

Josh

**Comment by lindbloom** [comment]
I'm fine with whatever your plan is for testing and development, I just think the link needs to go to "new"popr as that is what everyone should be using.
